### DISCUSSION TITLE
Research: Turret stabilization on cargo turrets.

### DISCUSSION THREAD

**Discussion by WitchFreya** [WitchFreya/SWSAddons#72] [Discovery]
For whatever reason pilot turret stabilization works fine but passenger does not.

It may not be possible without scripting.

Who knows. This'll need some Looking.

**Comment by EidolonForgery** [comment]
Need a vehicle where this is implemented and the bug can be properly examined.

**Reply by WitchFreya** [reply]
Define 'properly implemented'? As far as I'm aware, the only spots that I've seen turret stabilization work is on drones.

**Comment by WitchFreya** [comment]
Moved this to a discussion until we have proper AC or similar on it.
